charts/meteor-pipelines/templates/byon-validate-jupyterhub-image.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: byon-validate-jupyterhub-image
+spec:
+  params:
+   - name: url
+  results:
+   - name: python-runtime
+     description: Python runtime
+   - name: pip-packages
+     description: Packages installed in runtime
+   - name: message
+     description: Messages passed down from task
+  steps:
+    - name: jupyter-book-build
+      image: $(params.url)
+      securityContext:
+        privileged: true
```

Comment:
Hmmm... do we need that?

This made me realize that it's widespread across other tasks in the repo. I don't know what's the backround here though, will investigate more. But it would seem that this should work without additional privileges, no?

Reply:
@codificat  you are correct, I don't think privileged context is required.

Reply:
Good points thanks, fixed :+1: .